src/components/CreateList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import getToken from '../lib/tokens';
+
+const CreateList = () => {
+  const newList = () => {
+    const newToken = getToken();
+    localStorage.setItem('newToken', newToken);
+  };
+
+  return (
+    <form onClick={newList}>
+      <div>
+        <NavLink to="/list">
+          <button type="submit">Create List</button>
+        </NavLink>
+      </div>
+    </form>
```

Comment:
I think for a better semantic use of HTML, you can remove the `<form>` tag and have the `onClick` event on the actual `<button>`. Keeping `<button>` inside the `<div>` should be sufficient. :)